packages/core/content-manager/admin/src/preview/components/PreviewHeader.tsx

```diff
@@ -221,6 +221,7 @@ const PreviewHeader = () => {
       gridCols={3}
       paddingLeft={2}
       paddingRight={2}
+      style={{ gridTemplateRows: '47px' }}
```

Comment:
Where does 47px come from? Is this technique used elsewhere? I don't think we should be resorting to custom styles here, I think the DS should be able to handle this padding on its own

Reply:
lets discuss this tomorrow!  I agree we should find a way to make this be handled by the DS :) 

There were two options here, add a paddiing to the Header component (which was the orignal approach before this PR), or add no padding and use the Tabs component (if you check the design, it doesnt have any [padding](https://www.figma.com/design/BDCSSHnOZutJ4aaYTs41Bg/Preview?node-id=586-15081&node-type=canvas&t=G5Xjb3o7hmzuoyq5-0) )

Problem is D&P can be disabled and those tabs are not rendered, leavng the header without any padding (see Mark's [comment](https://github.com/strapi/strapi/pull/22025#issuecomment-2454293305))

To avoid using absolute positioining or negative paddings, we opted for this solution. But I would be glad to find something else!

Reply:
How about setting padding on the left and on the right separately?

Reply:
that we are already doing, but we also need vertical padding when there is no D&P tabs

Reply:
That's what I mean too, setting the vertical padding once for each section:

```jsx
<Grid paddingLeft={2} paddingRight={2}>
  <Flex paddingTop={1} paddingBottom={1}>
     // close btn + title + status
  </Flex>
  <MaybeTabs />
  <Flex paddingTop={1} paddingBottom={1}>
    // icon button
  </Flex>
  ```

Reply:
ah! 😄 that sounds like a good solution 🏃

Reply:
I think actually it might be 48px. It's in the design file for the height of the header. 

Since we are using grid here, I think grid-template-rows default value is auto and will take the height of its children. We only have one row and we want to set a fixed height.

Not sure why the prop isn't there as it is for gridCols, but we could easily do the same. My guess is we don't often need to explicitly set the height of rows in a grid.

https://github.com/strapi/design-system/blob/107f3b86f96a6dfb080a0d4fedca366ef895bd5c/packages/design-system/src/primitives/Grid/Grid.tsx#L32

I'm not as bothered by the custom style as I am about the style tag vs a styled component that could get the value from the theme. `sizes[9]` I think. 

https://github.com/strapi/design-system/blob/107f3b86f96a6dfb080a0d4fedca366ef895bd5c/packages/design-system/src/themes/common-theme.ts#L18

Reply:
> That's what I mean too, setting the vertical padding once for each section:
> 
> ```js
> <Grid paddingLeft={2} paddingRight={2}>
>   <Flex paddingTop={1} paddingBottom={1}>
>      // close btn + title + status
>   </Flex>
>   <MaybeTabs />
>   <Flex paddingTop={1} paddingBottom={1}>
>     // icon button
>   </Flex>
> ```

If this works, great, do that!

Reply:
Done! That seems to work ok

We are still not handling long title names, I was thinking of using a max width and overflowing the text with an ellipsis, I asked Lucas and we can handle that in another PR
![image](https://github.com/user-attachments/assets/c1ea33b9-ee78-4e48-8e9b-6bcc688b798f)

Reply:
Okay let's go ahead with this 🚂  Lucas gave me more context

Reply:
Hey guys, I just discussed with @Marc-Roig and we decided to simply set a 200px maxWidth with an ellipsis. As he said, no need to spend too much time on this for now 👍

> if they are not at the center of the screen, switiching between draft and publish would make the tabs move (if the title is different)

That's exactly why the tabs were originally centered, but I had too change this for the side editor iteration (that's a long story) 😄